Gemfile.lock

```diff
@@ -76,7 +71,7 @@ GEM
       rack (>= 1.0.0)
       rack-test (>= 0.5.4)
       xpath (~> 2.0)
-    celluloid (0.16.0)
+    celluloid (0.16.1)
```

Comment:
After my recent master rebase I tried to clean up `Gemfile.lock` file but apparently it got changed after `rake clean preview` again. So I am having some diffs here again... It seems I am confused about how I should handle `Gemfile.lock` file at all. It would be great if we flag is as `ignored` in git repo. How do you guys handle this file? You simply always exclude and changes in this file from all commits or what? I am not a Ruby expert so this is new to me.